db/migrations/02_add_desc_to_spaces.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE spaces ADD COLUMN description VARCHAR(200);
```

Comment:
We have also added a "02_.." migration file. Lets discuss whether this should just be added to the 01_create_spaces_table.sql file? The reason for this file is for end users to create required tables - its not clear they will need the incremental steps - rather just the end table requirements.